final_task/rss_reader/action_functions.py

```diff
@@ -72,12 +87,12 @@ def get_news(command_line_args, logger):
 
 def print_news_stdout(news_collection):
     """ Function for print news to stdout in text format. """
-    print("###############################################################")
-    print()
+    print("################################################################################")
+    print("")
```

Comment:
It is still better to look for a better solution than `print("")`

Reply:
fixed